src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -1320,9 +1347,16 @@ private static GencodeGtfExonFeature getExonWithinSpliceSiteWindow( final Varian
                                                                         final int spliceSiteVariantWindowBases ) {
         GencodeGtfExonFeature spliceSiteExon = null;
 
+        final int varStart = FuncotatorUtils.getIndelAdjustedAlleleChangeStartPosition(variant);
+        final int varEnd = variant.getEnd();
```

Comment:
Why don't you repeat the logic above to set the end here?

Reply:
You're absolutely right - I need to update it.

I missed it because I was not using intervals to do the boundary checks.